tools/define-compose-url.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -euo pipefail
 
-curl -L http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.4/COMPOSE_ID > COMPOSE_ID
+# jq is needed to get COMPOSE_ID
+sudo dnf -y install jq
+
+# This will get the latest finished compose that was tagged accordingly
+curl -kL "https://cts.engineering.redhat.com/api/1/composes/?release_version=8.5.0&tag=nightly" | jq -r .items[0].compose_info.payload.compose.id > COMPOSE_ID
```

Comment:
The reason I wanted to introduce this is because it should always point to the latest RTT_ACCEPTED nightly. It can happen that the latest 8.5 folder in internal download servers can contain an unfinished compose or otherwise broken or I encountered it in the past at least... But like you're pointing out it's probably not that reliable as the RHEL-8.5.0-20210622.n.0 was also RTT_ACCEPTED... 

I agree to just drop it and point to latest 8.5... Also I noticed a latest finished 8.5 symlink exists, might be a bit better.

IMO it would be best if we could just trigger on some UMB message that a new nightly suitable for testing has been created but I guess that can be solved later.

Reply:
+1 for the `-finished` symlink. I didn't know about it.